transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -162,6 +161,13 @@ def forward(
         with_input_all_gather = parallel_mode == "column" and sequence_parallel
 
         # Configure Userbuffers communication (comm+GEMM overlap)
+        if debug:  # turn off userbuffers in debug mode
+            ub_overlap_ag_fprop = False
+            ub_overlap_rs_fprop = False
+            ub_overlap_ag_dgrad = False
+            ub_overlap_rs_dgrad = False
+            ub_bulk_wgrad = False
+            ub_bulk_dgrad = False
```

Comment:
Is this just because our naming in the autograd functions of different modules is inconsistent?

Reply:
No, the question is why you changed the previous code which had a function call here to turn off unsupported features during debug into doing it manually in every module.